bindings/java/README.md

```diff
@@ -62,14 +62,48 @@ You can use Maven to build both Rust dynamic lib and JAR files with one command
 ./mvnw clean package -DskipTests=true
 ```
 
+## Setup Tests
+
+Please copy `{project.rootdir}/.env.example` to `src/test/resources/.env` and change the values on need.
```

Comment:
Hi, I apologize for not explaining my idea clearly. My expectation is that all projects can utilize the same `{project.rootdir}/.env` file, eliminating the need for users to manage separate environment setups for different modules.

```suggestion
Please copy `{project.rootdir}/.env.example` to `{project.rootdir}/.env` and change the values on need.
```

Is this possible?

Reply:
Okay, I got it. I'll try.🫡